examples/CMakeLists.txt

```diff
@@ -17,6 +17,13 @@ set(ASYNC_SIMPLE_SOURCE_FILES
 add_executable(async_simple ${ASYNC_SIMPLE_SOURCE_FILES})
```

Comment:
I've added a lot of single comments, please take a look

Reply:
Thank you for the comments! I think I got everything. I wasn't sure if it was better to create a new commit or amend and I ended up creating a new commit. In retrospect, it probably should have just been an amend. If it's difficult to read, please let me know and I'll fix it.